client/blocks/importer/wordpress/upgrade-plan/skeleton/index.tsx

```diff
@@ -1,19 +1,96 @@
+import type { FC } from 'react';
 import './style.scss';
 
-interface SkeletonProps {
-	width?: number | string;
-	height?: number | string;
-}
-
-// Borrowed from site-profiler.
-export const Skeleton = ( { width = '100%', height = 20 }: SkeletonProps ) => {
+export const Skeleton: FC = () => {
 	return (
-		<div
-			className="import-upgrade-plan-skeleton"
-			style={ {
-				width,
-				height,
-			} }
-		/>
+		<div className="import__upgrade-plan-details import__upgrade-plan-details--loading">
```

Comment:
Some wrapper elements were duplicated from the original components in order to simulate the columns.

The other way would be to add the Skeleton inside the components, but it would make it spread and demand a big refactor that wouldn't be worth it IMO.